### PR TITLE
Feature: fundraisers

### PR DESCRIPTION
**Pull request by kangarang**
initial work to add page for fundraisers: base components for generating pages per fundraiser (from json data) and storybook improvements

### CODE REVIEW

File: gatsby-site/src/components/Fundraiser/FundraiserForm.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+          />
+
+          <FieldText
+            type="text"
+            name="firstName"
+            id="pledge-first-name"
+            label="First Name"
+            placeholder="Enter your first name"
+            value={values.firstName}
+            required
```
Comment (akuanti) [inline gatsby-site/src/components/Fundraiser/FundraiserForm.js]: This should set `onChange`, right?

File: gatsby-site/src/components/Fundraiser/FundraiserOverview.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            <ReasonToContribute iconSrc={flag} title="Prioritized Grants">
+              When you donate to Vivek Singh’s fundraiser, the Gitcoin team moves up Panvala’s
+              leaderboard for grant application priority. View the leaderboard.
+            </ReasonToContribute>
+
+            <ReasonToContribute iconSrc={support} title="Show Your Support">
+              Your name will be listed on this page as Vivek Singh's supporter. Our whole community
+              values our donors!
+            </ReasonToContribute>
+          </Box>
```
Comment (akuanti) [inline gatsby-site/src/components/Fundraiser/FundraiserOverview.js]: These should dynamically render the fundraiser's name.

File: gatsby-site/src/components/Fundraiser/FundraiserProfile.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  return (
+    <Box mt="-5vw" className="bottom-clip-down relative z-3" bg="white" height="1000px">
+      <Box p={'10vw'} flex column>
+        <Box>
+          <img src={vivekS} />
+        </Box>
+        <ProfileCopy {...props} />
+      </Box>
+
+      <ProfileLink />
```
Comment (akuanti) [inline gatsby-site/src/components/Fundraiser/FundraiserProfile.js]: I think this link only belongs on the `FunderBoard`.

**Comment by kangarang** [comment]
> Currently this uses the fundraiser's first name for the URL. We should create a different field to identify the fundraiser and use that to generate the URL.

what is the reasoning for not using first name?

**Comment by akuanti** [comment]
John Smith and John Jones can't both have fundraiser pages if we use the first name.

**Pull request merged**